pages/index.tsx

```diff
@@ -199,11 +200,11 @@ export const HomePage: NextPage = () => {
           />
         </InputGroup>
       </Stack>
-      <Stack spacing="32px" px="112px" py="64px">
+      <Stack spacing="32px" px="10%" py="64px" minWidth="800px">
         <Button onClick={onDrawerOpen}>View your list</Button>
         <Flex w="100%">
           <Flex w="50%">
-            <Box w="50%" pr="20px">
+            <Box w="50%" pr="20px" minWidth="250px">
```

Comment:
I don't love just hardcoding a `minWidth` on all these selects to make them fit on the screen - I feel like using Chakra's `Stack` and/or `Grid` component would be a better solution?